src/__test__/components/MetricDetails/MetricDetails.spec.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { render  } from '@testing-library/react';
+
+import { mockData, district, metric, metricBar } from './mockData';
+import MetricDetails from '@components/MetricDetails';
+
+jest.mock('../../../data/Data.json', () => (mockData));
+
+
+describe('MetricDetails component', () => {
+  it('should be able to render the MetricDetails component without bar', () => {
+    const { getByText } = render(
+      <MetricDetails district={district} metric={metric} />,
+    );
+
+    const metricValue = getByText('1035544.91');
+    expect(metricValue).toBeTruthy();
```

Comment:
Here we have a double verification, since we are using the testing-library we just need to use the functions from there and we already have the collection, verification and the throw error. 

https://testing-library.com/docs/queries/about/